src/Keboola/Console/Command/AllStacksIterator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Keboola\Console\Command;
+
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\StringInput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class AllStacksIterator extends Command
+{
+    public const ARG_COMMAND = 'command-to-run';
+    public const ARG_PARAMS = 'params';
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('manage:call-on-stacks')
+            ->addArgument(self::ARG_COMMAND, InputArgument::REQUIRED, 'command')
+            ->addArgument(self::ARG_PARAMS, InputArgument::REQUIRED, 'params');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): ?int
+    {
+        $commandName = $input->getArgument(self::ARG_COMMAND);
+        $command = $this->getApplication()->find($commandName);
+        $cmndInput = $input->getArgument(self::ARG_PARAMS);
+
+        $stacksFile = file_get_contents('http-client.env.json');
+        $stacksTokensFile = file_get_contents('http-client.private.env.json');
+
+        if (!$stacksFile || !$stacksTokensFile) {
+            throw new Exception('Input http-client files are not available');
+        }
+
+        $stacks = json_decode($stacksFile, true);
+        $tokens = json_decode($stacksTokensFile, true);
+
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+        // iterates over all stacks
+        foreach ($stacks as $stackName => $stack) {
+            if (!isset($tokens[$stackName]) || $tokens[$stackName] === '') {
+                $output->writeln(sprintf('Token for %s not found or it is empty. Skipping', $stackName));
+                continue;
+            }
+
+            $token = $tokens[$stackName];
+
+            // build input for the target command. It adds token and host from http-client files.
+            $inputForThisStack = sprintf('%s %s %s %s', $commandName, $token['manageToken'], $stack['host'], $cmndInput);
```

Comment:
Tady mi přijde akorát škoda to dělat takhle a udělal bych to tak, že bych to přímo naplnil do \Symfony\Component\Console\Input\ArrayInput - tady bych si nebyl jistej tím, jestli je to nějak správně escapované.

Reply:
Ale není to úplně triviální, takže jen nápad na budoucí vylepšení...